File: CS50x - Intro to CompSci/sentimental-mario-more/mario.py
def heightI():
    h = input("Height: ")
    try:
        h = int(h)
    except:
        return heightI()
    else:
        if h <= 8 and h >= 1:
            return h
        else:
            return heightI()


while True:
    height = heightI()
    if height <= 8 and height >= 1:
        break
for row in range(height):
    for column in range(2 * height + 2):
        if column < height - row - 1 or column == height or column == height + 1:
            print(" ", end="")
        elif (column >= height - row - 1 and column < height) or (column <= height + row + 2):
            print("#", end="")
    print()

File: cs50/x/mario-more/mario.c
// Include libraries
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Get height
    long height;
    do
    {
        height = get_long("Height: ");
    }
    while (height > 8 || height < 1);

    // Print pryamids
    for (long row = 0; row < height; row++)
    {
        for (long column = 0; column < 2 * height + 2; column++)
        {
            if (column < height - row - 1 || column == height || column == height + 1)
            {
                printf(" ");
            }
            else if ((column >= height - row - 1 && column < height) || (column <= height + row + 2))
            {
                printf("#");
            }
        }
        printf("\n");
    }
}